libparsec/crates/client/tests/unit/certif/add_shamir_recovery_brief_certificate.rs

```diff
@@ -0,0 +1,373 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec_tests_fixtures::prelude::*;
+
+use crate::certif::{
+    CertifAddCertificatesBatchError, InvalidCertificateError, MaybeRedactedSwitch,
+};
+
+use super::utils::certificates_ops_factory;
+
+#[parsec_test(testbed = "minimal")]
+async fn ok(env: &TestbedEnv) {
+    let env = env.customize(|builder| {
+        builder.new_user("bob");
+
+        builder.new_shamir_recovery(
+            "bob",
+            1,
+            [("alice".parse().unwrap(), 1.try_into().unwrap())],
+        );
+    });
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    // To test `shamir_recovery_brief_certificate`, we remove the `shamir_recovery_share_certificate`
+    // which is tested in another file.
+    let mut shamir_recovery_certificates = env.get_shamir_recovery_certificates_signed();
+    shamir_recovery_certificates.pop();
+
+    let switch = ops
+        .add_certificates_batch(
+            &env.get_common_certificates_signed(),
+            &[],
+            &shamir_recovery_certificates,
+            &Default::default(),
+        )
+        .await
+        .unwrap();
+
+    p_assert_matches!(switch, MaybeRedactedSwitch::NoSwitch)
+}
+
+#[parsec_test(testbed = "minimal")]
+async fn multiple(env: &TestbedEnv) {
+    let env = env.customize(|builder| {
+        builder.new_user("bob");
+
+        builder.new_shamir_recovery(
+            "bob",
+            1,
+            [("alice".parse().unwrap(), 1.try_into().unwrap())],
+        );
+
+        builder.new_shamir_recovery(
+            "bob",
+            1,
+            [("alice".parse().unwrap(), 1.try_into().unwrap())],
+        );
+    });
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    // To test `shamir_recovery_brief_certificate`, we remove the `shamir_recovery_share_certificate`
+    // which is tested in another file.
+    let mut shamir_recovery_certificates = env.get_shamir_recovery_certificates_signed();
+    shamir_recovery_certificates.pop();
```

Comment:
Since you create 2 shamir recory, 

Why you don't do 2 `pop` to remove both share of both operations ?

Reply:
Done